### ISSUE TITLE
Hovercar with indestructible option should be truly indestructible

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#1556]
**Feature description**
If a hovercar is made indestructible through the context menu, it should be _**indestructible**_

**Feature purpose**
It does not make sense that it gets damaged if it is indestructible

**Additional context**
If #1555 gets added this should apply to the other vehicles too

**Comment by ThePeashooter23** [comment]
I agree, but this should be a labeled as a bug instead of a feature request. Since you are reporting a thing in the game that isn't doing what's intended, it's a bug.
